features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,17 @@
-# put step definitions in this file
\ No newline at end of file
+# put step definitions in this file
+Given(/^I visit the site$/) do
+  visit '/'
+end
+
+Then(/^viewing the text "([^"]*)"$/) do |your_name|
```

Comment:
I commented on this already, but step that assert that something has happened should reflect the intention to assert. Perhaps use wording like: `I should see...` or similar?